app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    peoples_objects = [Person(human["name"], human["age"]) for human in people]
+
+    for human in people:
+
+        if human.get("wife"):
+            wife = Person.people[human["wife"]]
+            Person.people[human["name"]].wife = wife
```

Comment:
```suggestion
        if wife:=human.get("wife"):
            Person.people[human["name"]].wife = Person.people[wife]
```

Reply:
But I can't use it, because Python gives me mark: "invalid syntax". I using 3.7 Python